api/src/test/scala/MacroSpec.scala

```diff
@@ -283,6 +283,23 @@ final class MacroSpec extends org.specs2.mutable.Specification {
       }
     }
 
+    "handle union types without sealed trait" in {
```

Comment:
Can be moved up (in the `>>` group) without repeating `handle union types` in description

Reply:
Done